README.md

```diff
@@ -1,11 +1,187 @@
+Your bashrc is not configured. To ensure bashrc is being utilized correctly, run:
+```
+$echo PATH
+```
+
+#### Angular 404 Message
+Try running the following: 
+```
+python manage.py bower install
+```
+
+Make sure you select "d3#^3.4.4 which resolved to 3.5.17 and is required by nvd3#1.8.5"!
+
+#### Egg info failed error
+May be specific to what dependency is being downloaded, but if for “postgresql”, brew install it instead with: 
+```
+brew install postgresql
+```
+
+#### 'Module_six_moves_urllib_parse' object has no attribute 'urlparse'
+According to [Google](https://developers.google.com/google-apps/tasks/quickstart/python), 'this error can occur in Mac OSX where the default installation of the "six" module (a dependency of this library) is loaded before the one that pip installed.' There are a variety of different methods to fix this, and you can try any or all of the following methods:
+
+- Add pip's install location to the PYTHONPATH system environment variable:
+	
+	Determine pip's install location with the following command:
+	```
+	pip show six | grep "Location:" | cut -d " " -f2
+	```
+	
+	Add the following line to your ~/.bashrc file, replacing <pip_install_path> with the value determined above:
+	```
+	export PYTHONPATH=$PYTHONPATH:<pip_install_path>
+	```
+	
+	Reload your ~/.bashrc file in any open terminal windows using the following command
+	```
+	source ~/.bashrc
+	```
+	
+	If this doesn't work, try running the command again, or editing the pip_install_path then running the command again, this value might change. (Example of one that works: export PYTHONPATH=/Users/tiffanyqi/Desktop/env/env/lib/python2.7/site-packages)
+
+- Uninstall and reinstall six by running
+	```
+	sudo pip uninstall six
+	sudo pip install six
+	```
+
+- Uninstall and reinstall [python2.7](http://stackoverflow.com/questions/34303294/how-to-fix-broken-python-2-7-11-after-osx-updates)
+
+- Uninstall and reinstall Calendar Analytics altogether
+
+- DO NOT DOWNGRADE to google-api-python-client 1.3.2 (will throw another kind of error)
+
+#### Category20 undefined
+Run the following: 
+```
+python manage.py bower update
+```
+
+Be sure to choose “d3#^3.4.4 which resolved to 3.5.17 and is required by nvd3#1.8.5”!
+
+### Another issue?
+Feel free to add a issue to this repo, and someone will get back to you!
+
+
+## Built on
+- Django
+- Angular
+- Postgresql
+- Angular-NVD3
+- FullCalendar
+- Angular UI Calendar
+- JQuery - Ajax
+- Moment
+- Google Auth
+- Bower
+
+## Our Team
+- [Max Fang](http://maxfangx.com/): Full-Stack Developer, Creator
+- [Tiffany Qi](http://tiffanyqi.com/): Product Manager, Front-End Developer
+- [James Uejio](http://jamesuejio.com/): Full-Stack Developer
+- [Cindy Tung](https://www.linkedin.com/in/cindytung96): Full-Stack Developer
+- [David Nguyen](https://www.linkedin.com/in/dnguyenv): Visual Designer
+- [Andrew Huang](http://andrewhuang.tk): Full-Stack Developer
+
+## License
+The MIT License (MIT)
```

Comment:
Is there any potential for this to become a legal issue in the event that we want to privatize our code and turn our product into a startup?